build-zip.sh

```diff
@@ -1,20 +1,38 @@
 #!/bin/bash -e
-BUILD_NUMBER=$1;
+BUILD_NUMBER=${1:-"dev"}
+ROOT_DIR=$(pwd)
+rm -rf ./build/*
 mkdir -p build/organic
 ls -alh
-sudo chown -R runner:docker .
+
+if ! [[ $BUILD_NUMBER == "dev" ]]; then
+  sudo chown -R runner:docker .
+fi
+
 cp -r Organic build/organic/
 cp organic.php build/organic/
 cp readme.txt build/organic/
-
-sed -i "s/ORGANIC_PLUGIN_VERSION_VALUE/${BUILD_NUMBER}/" build/organic/organic.php build/organic/readme.txt
+# macos sed has different options
+if [[ $(uname) == 'Darwin' ]]; then
+  sed -i '' "s/ORGANIC_PLUGIN_VERSION_VALUE/${BUILD_NUMBER}/" build/organic/organic.php build/organic/readme.txt
+else
+  sed -i "s/ORGANIC_PLUGIN_VERSION_VALUE/${BUILD_NUMBER}/" build/organic/organic.php build/organic/readme.txt
+fi
 
 cp composer.json build/organic/
 cp composer.lock build/organic/
-cp -r vendor build/organic/
+
+cd build/organic
+composer install --no-dev
+cd $ROOT_DIR
+
+# cp -r vendor build/organic/
 (cd affiliate/ && npm ci && npm run build)
-mkdir -p build/organic/affiliate/
-cp -r affiliate/blocks/ build/organic/affiliate/
+mkdir -p build/organic/affiliate/productCard build/organic/affiliate/productCarousel
+cp -r affiliate/blocks/productCard/build build/organic/affiliate/productCard/
+cp affiliate/blocks/productCard/block.json build/organic/affiliate/productCard/
+cp -r affiliate/blocks/productCarousel/build build/organic/affiliate/productCarousel/
+cp affiliate/blocks/productCarousel/block.json build/organic/affiliate/productCarousel/
```

Comment:
I don't think this will work. See https://organicly.atlassian.net/browse/EP-2249?focusedCommentId=58233. We register scripts based on the affiliate/blocks path [here](https://github.com/orgnc/wordpress-plugin/blob/9d1afb7bd0932f7f69f8b0412858c56862bd2fac/Organic/Affiliate.php#L21-L32), which looks like it wasn't changed in this PR. Is there a reason we need to modify this portion of the build file?